src/classes/Election.ts

```diff
@@ -103,21 +103,58 @@ export class Election {
     return this.contestNames;
   }
 
-  printContestNames() {
-    this.contests.forEach((element) => {
+  createCVR() {
```

Comment:
_suggestion_ Constructing a JSON string like this is brittle, and difficult to extend if you need to make changes or additions to it. Further, it's not really possible to see what the shape of the JSON looks like, which prevents us from knowing whether it conforms to any specification unless you run the code and manually confirm.

Instead, I would suggest: 

1. Write an interface for what this JSON object should look like
2. In this function, create a JSON object which conforms to that interface and has all the values necessary
3. Use `JSON.stringify(cvr)` to convert the object to JSON

What do you think?